system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -573,6 +573,8 @@
                                                       (data-umm-c/collection {:ShortName "coll1"
                                                                               :EntryTitle "entry-title1"})
                                                       {:token "mock-echo-system-token"})
+          fake-concept (subscription-util/make-subscription-concept {:SubscriberId "user1"
```

Comment:
This kind of error where the collection concept id is not even valid should have been caught by subscription validation code.

Reply:
ticket filed for this.